packages/blocks/src/components/portal.ts

```diff
@@ -41,6 +45,34 @@ export class Portal extends LitElement {
   }
 }
 
+/**
+ * Similar to `<affine-portal>`, but only renders once when called.
+ *
+ * The template should be a **static** template since it will not be re-rendered.
+ *
+ * See {@link Portal} for more details.
+ */
+export function createLitPortal({
```

Comment:
I think we can make the `affine-portal` classname as part of options, and move this into `@blocksuite/lit`